components/config-mgmt-service/backend/elastic/suggestions.go

```diff
@@ -247,12 +247,7 @@ func (es Backend) getArrayAggSuggestions(term string, text string, filters map[s
 	suggs := make([]string, 0)
 	if aggResult != nil {
 		for _, bucket := range aggResult.Buckets {
-			// When elasticsearch find a match in an array it returns that whole array and includes other values in that array as buckets.
-			// Because of this we will filter any buckets that do not contain any of the search string text
-			bucketname := string(bucket.KeyNumber)
-			if strings.Contains(bucketname, textLower) {
```

Comment:
This was removed because we are doing the filtering above with regex in elasticsearch.